compose/composer.go

```diff
@@ -4,14 +4,64 @@ import (
 	"bytes"
 )
 
-// Deprecated, see v2_composer
-func TransactionRequest(numTransactions int, numSpans int, numFrames int) []byte {
-	return nil
+func Compose(numErrors, numTransactions, numSpans, numFrames int) []byte {
+	//TODO: shuffle body except for metadata
```

Comment:
do you mind removing todos and create v2 issue similar to this one https://github.com/elastic/hey-apm/issues/30?

Reply:
https://github.com/elastic/hey-apm/issues/54